content/logs/aws.md

```diff
@@ -8,50 +8,81 @@ beta: true
+5. Under Policy templates, search for and select **s3 object read-only permissions.**
+6. Select **Create Function.**
     {{< img src="logs/aws/author_from_scratch.png" alt="Author from Scratch" responsive="true" >}}
-3. For `Role`, select **Create new role from template(s)** and give the role a unique name
-4. Under Policy templates, search for and select **s3 object read-only permissions**
-    {{< img src="logs/aws/basic_information.png" alt="Basic Information" responsive="true" >}}
-5. Select **Create Function**
 
-### Provide the code and configure the lambda
+### Provide the code and configure the Lambda
 
 1. Copy and paste the code from [this repo](https://github.com/DataDog/dd-aws-lambda-functions/blob/master/Log/lambda_function.py) into the function code area.
-2. Change the Runtime to **Python 2.7** and set the handler to **lambda_function.lambda_handler**
+2. Ensure the Handler reads **lambda_function.lambda_handler**
     {{< img src="logs/aws/select_python.png" alt="Select Python" responsive="true" >}}
 3. At the top of the script you'll find a section called `#Parameters`. You have two options for providing the API Key that the Lambda function requires:
     
     * Setup an environment variable (Preferred)
     * Edit the code directly with your Datadog API Key
     {{< img src="logs/aws/dd_api_key_setup.png" alt="DD API key setup" responsive="true" >}}
 
-4. Set the memory to the highest possible value.
-5. Set the timeout limit. We recommend **120 seconds**
+4. Scroll down beyond the inline code area to **Basic Settings**.
+5. Set the memory to the highest possible value.
+6. Set the timeout limit. We recommend **120 seconds.**
     {{< img src="logs/aws/basic_settings.png" alt="Basic Settings" responsive="true" >}}
-6. Scroll back to the top of the page and hit **Save and test**
+7. Scroll back to the top of the page and hit **Save.**
 
 ### Test your Lambda
 
-1. Configure the test event. Select **Cloudwatch Logs** as the sample event.
+1. Press **Test**. 
+2. Search for and select **Cloudwatch Logs** as the sample event.
     {{< img src="logs/aws/test_event.png" alt="Test Event" responsive="true" >}}
-2. Save and test.
+2. Give the event a unique name and press **Create**.
+3. Press Test and ensure the test passes with no errors.
 
 ## Collection
 
-Your lambda function is now ready to send logs to your Datadog platform.
-Setup the relevant triggers for each AWS service you want to monitor.
-
-### S3, Cloudwatch, API Gateway, Kinesis and SNS
-In your lambda, go in the triggers tab and select `Add Trigger`:
+Your Lambda function is now ready to send logs to the Datadog platform. Setup the relevant triggers for each AWS service you want to monitor.
+
+**The Lambda function you just created must be in the same region as the S3 bucket you are using as a trigger. If you have logs across multiple regions you must create additional Lambda funcions.**
+
```

Comment:
@DanielLanger I would add here a small paragraph that explains that there is two way to configure the triggers:
automatic (which requires some permissions ) and manual.

With a link on automatic and manual to the corresponding section (and maybe a link on permissions as well).
This way it might be easier when we will split this document and clearer for users as they are directly aware of the two choices.

Reply:
And i would add the exact same paragraph on the onboarding with the same links.